open-api.yaml

```diff
@@ -2325,4 +2356,102 @@ paths:
                     pkgVersion: 0.21.0
         '401':
           $ref: '#/components/responses/401'
+  /tasks:
+    get:
+      summary: Get all tasks
+      description: 'Get all [tasks](https://docs.meilisearch.com/learn/advanced/asynchronous_tasks.html)'
+      tags:
+        - tasks
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/task'
+              examples:
+                Example:
+                  value:
+                    data:
+                      - uid: 1
+                        indexUid: movies
+                        status: succeeded
+                        type: documentsAddition
+                        details:
+                          numberOfDocuments: 79000
+                        duration: PT1S
+                        enqueuedAt: '2021-01-01T09:39:00.000000Z'
+                        startedAt: '2021-01-01T09:39:01.000000Z'
+                        finishedAt: '2021-01-01T09:39:02.000000Z'
+                      - uid: 0
+                        indexUid: movies_Review
+                        status: failed
+                        type: documentsAddition
+                        details:
+                          numberOfDocuments: 67493
+                        error:
+                          message: missing primary key
+                          errorCode: missing_primary_key
+                          errorType: invalid_request_error
+                          errorLink: 'https://docs.meilisearch.com/errors#missing_primary_key'
+                        duration: PT5S
+                        enqueuedAt: '2021-01-01T09:38:00.000000Z'
+                        startedAt: '2021-01-01T09:38:02.000000Z'
+                        finishedAt: '2021-01-01T09:38:07.000000Z'
+      operationId: tasks.list
+      parameters: []
+      security:
+        - apiKey: []
+  '/tasks/:taskUid':
+    get:
+      summary: Get a task
```

Comment:
Should we say `get a task status` ? As `get a task` means `get something to do`.

Reply:
The open API spec is also driven by what the documentation decides in terms of text or explanation, that's why I'm taking up the texts later. The most important thing is to focus on the structure changes more than the text